src/utils/getCalendarDaySettings.js

```diff
@@ -36,7 +38,13 @@ export default function getCalendarDaySettings(day, ariaLabelFormat, daySize, mo
 
   let ariaLabel = getPhrase(chooseAvailableDate, formattedDate);
   if (selected) {
-    ariaLabel = getPhrase(dateIsSelected, formattedDate);
+    if (modifiers.has('selected-start')) {
```

Comment:
Should I add a null check here for `dateIsSelectedAsCheckin` and `dateIsSelectedAsCheckout`, so that it doesn't throw an error if someone using react-dates is overriding the default phrases and hasn't added these two phrases?

Reply:
That might be a good mitigating factor.